packages/fxa-settings/src/react-app-env.d.ts

```diff
@@ -7,18 +7,3 @@
 type hexstring = string;
 
 type Hash<T> = { [key: string]: T };
-
-interface FlowQueryParams {
```

Comment:
I moved this out of this file because it seems it should be mostly left alone except for declaring app-global types/modules.